server/queue/queue-api/src/main/java/org/apache/james/queue/api/MailQueueFactory.java

```diff
@@ -42,5 +42,5 @@
 
     T createQueue(String name);
 
-    Set<T> listCreatedMailQueues();
+    Set<String> listCreatedMailQueues();

```

Comment:
Can we have a MailQueueName POJO? Strong typing in code APIs sounds important to me.

Reply:
I will do.